docker-compose.yml

```
@@ -30,6 +30,30 @@ services:
       # - ./.env
       # - ./frontend/.env
 
+  ########################################################
+  # ADMIN INTERFACE ######################################
+  ########################################################
+  admin:
+    image: gradido/admin:latest
+    build:
+      context: ./admin
+      target: production
+    networks:
+      - internal-net
```

this requires `external-net`also in production, since its a frontend, correct?

```suggestion
      - internal-net
      - external-net
```